Doc/library/tarfile.rst

```diff
@@ -104,7 +104,7 @@ Some facts and figures:
    | ``'w'`` or       | Open for uncompressed writing.              |
    | ``'w:'``         |                                             |
    +------------------+---------------------------------------------+
-   | ``'w:gz'``       | Open for gzip compressed writing.           |
+   | ``'w:gz'``       | Open for gzip-compressed writing.           |
```

Comment:
Why are you hyphenating these? The following line has bzip2 compressed without hyphens.